Second commit useful in all version, but PHP < 7.3 strangely don't raise these warnings

Example:
```
/dev/shm/BUILD/php73-php-pecl-request-1.0.0/NTS/serverresponse.c:588:54: warning: field precision specifier '.*' expects argument of type 'int', but argument 4 has type 'size_t {aka long unsigned int}' [-Wformat=]
                             "JSON encoding failed: %.*s", Z_STRLEN(json_errmsg), Z_STRVAL(json_errmsg));

```

--- a/serverrequest.c
+++ b/serverrequest.c
@@ -672,12 +672,12 @@ static inline void server_request_set_url(zval *object, zval *server)
     // Form array
     array_init_size(&arr, 8);
     if( url->scheme ) {
-        add_assoc_string(&arr, "scheme", ZSTR_VAL(url->scheme));
+        add_assoc_string(&arr, "scheme", URL_STR(url->scheme));
     } else {
         add_assoc_null(&arr, "scheme");
     }
     if( url->host ) {
-        add_assoc_string(&arr, "host", ZSTR_VAL(url->host));
+        add_assoc_string(&arr, "host", URL_STR(url->host));
     } else {
         add_assoc_null(&arr, "host");
     }
@@ -687,27 +687,27 @@ static inline void server_request_set_url(zval *object, zval *server)
         add_assoc_null(&arr, "port");
     }
     if( url->user ) {
-        add_assoc_string(&arr, "user", ZSTR_VAL(url->user));
+        add_assoc_string(&arr, "user", URL_STR(url->user));
     } else {
         add_assoc_null(&arr, "user");
     }
     if( url->pass ) {
-        add_assoc_string(&arr, "pass", ZSTR_VAL(url->pass));
+        add_assoc_string(&arr, "pass", URL_STR(url->pass));
     } else {
         add_assoc_null(&arr, "pass");
     }
     if( url->path ) {
-        add_assoc_string(&arr, "path", ZSTR_VAL(url->path));
+        add_assoc_string(&arr, "path", URL_STR(url->path));
     } else {
         add_assoc_null(&arr, "path");
     }
     if( url->query ) {
-        add_assoc_string(&arr, "query", ZSTR_VAL(url->query));
+        add_assoc_string(&arr, "query", URL_STR(url->query));
     } else {
         add_assoc_null(&arr, "query");
     }
     if( url->fragment ) {
-        add_assoc_string(&arr, "fragment", ZSTR_VAL(url->fragment));
+        add_assoc_string(&arr, "fragment", URL_STR(url->fragment));
     } else {
         add_assoc_null(&arr, "fragment");
     }

